src/libs/blueprint/conduit_blueprint_mpi_mesh.hpp

```diff
@@ -66,6 +66,11 @@ void CONDUIT_BLUEPRINT_API generate_index(const conduit::Node &mesh,
                                           Node &index_out,
                                           MPI_Comm comm);
 
+//-------------------------------------------------------------------------
+void CONDUIT_BLUEPRINT_API generate_partition(const conduit::Node &mesh,
```

Comment:
I'd like to see this changed to `generate_domain_to_rank_map`, as per @cyrush's suggestion. What are your thoughts, @nselliott?

Reply:
Yes, that's fine.  Tells exactly what it is.